packages/back-end/types/experiment-snapshot.d.ts

```diff
@@ -192,15 +192,13 @@ export interface ExperimentSnapshotTrafficDimension {
 // Params for gbstats
 export interface ExperimentMetricAnalysisParams {
   variations: ExperimentReportVariation[];
-  metric: ExperimentMetricInterface;
-  rows: ExperimentMetricQueryResponseRows;
-  dimension: string | null;
-  baselineVariationIndex: number;
-  differenceType: DifferenceType;
   phaseLengthHours: number;
   coverage: number;
-  statsEngine: StatsEngine;
-  sequentialTestingEnabled: boolean;
-  sequentialTestingTuningParameter: number;
-  pValueThreshold: number;
+
+  analyses: ExperimentSnapshotAnalysisSettings[];
```

Comment:
Refactored to just pass down the settings here again. No reason not to, honestly. It's basically all the same fields, but some of them are optional. But since we use the fallbacks anyways and make them required in the new `AnalysisSettingsForStatsEngine` anyways, just reusing `ExperimentSnapshotAnalysisSettings` cuts out a third analysis settings interface that wasn't doing much.